clusters/script-gen-readonly-kubeconfig.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+cat <<EOF | kubectl apply -f -
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: crole-customresources-readyonly
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
```

Comment:
it aggregates using this label to `view`

https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles

Reply:
That makes sense, good find!